[IMP] Option to define the same Decoration Types (@DecoTypeID)

psugne opened the pull request:
either at the product level or per individual decoration

https://hive.versada.eu/wp/25129

paulius-sladkevicius commented:
@psugne could you review all README as looks it's quite outdated with links to schema and other parts as ":Version: 1.1 of 2023-09-06", authors..

psugne committed:
[IMP] Option to define the same Decoration Types (@DecoTypeID)

either at the product level or per individual decoration

https://hive.versada.eu/wp/25129
